packages/cra-template-imodeljs/template/.env

```diff
@@ -0,0 +1,20 @@
+# -----------------------------------------------------------------------------------------------------------
+# Project and iModel (REQUIRED)
+# Set these variables before testing - create a new project and
+# iModel with the developer registration procedure here - https://git.io/fx8YP
+# -----------------------------------------------------------------------------------------------------------
+
```

Comment:
How does this variable help?

Reply:
Try creating a new app with the template. By default, when you start it up, it complains about the version of babel being used, this will ignore that. https://dev.to/stephencweiss/debugging-create-react-app-56ob